### ISSUE TITLE
Incorrect matrix in beginner tutorial on www.tensorflow.org

### ISSUE THREAD

**Issue by medakk** [tensorflow/tensorflow#185]
The following image on the [Beginner's tutorial page](http://www.tensorflow.org/tutorials/mnist/beginners/index.md) is wrong:
![softmax-regression-scalarequation](https://cloud.githubusercontent.com/assets/10536745/11130763/d030c652-89ac-11e5-8d19-186bf9af9b28.png)

Here is a corrected version:
![softmax-regression-scalarequationcorrected](https://cloud.githubusercontent.com/assets/10536745/11130892/910fafaa-89ad-11e5-88d0-083f3da5a3fe.png)

**Comment by vrv** [comment]
(This has been fixed already in our repo, pending a push to the website)
